src/ImageSharp/Formats/Tiff/TiffEncoderEntriesCollector.cs

```diff
@@ -274,6 +278,7 @@ public void Process(TiffEncoderCore encoder)
                     Value = (ushort)encoder.PhotometricInterpretation
                 };
 
+                this.collector.AddOrReplace(planarConfig);
```

Comment:
note that I have explicitly added the chunky planar configuration, otherwise it could have accidentally be possible that it would be set to planar, if the decoded images was planar.

Reply:
I don't really understand this change. Could you please explain to me what the issue is with pulling the value from the decoder if present?

Reply:
There are two possible values of the `PlanarConfiguration`:
- *Chunky*: The data is stored as RGBRGBRGB. This is the default mode.
- *Planar*: This is more uncommon. The data is stored as: 
RRR
GGG
BBB

The Tiff Encoder does always encodes the data as `Chunky`, but does not make this explicitly by storing a value for the `PlanarConfiguration`. The value is omitted and because decoders will assume `Chunky` this will work. 

Now in the case of when we decode an actual 'Planar' Tiff and the reencode this TIFF again, the `PlanarConfiguration` would carry over to the encoded image, because its present in the EXIF profile. This would change the interpretation of the data and make the encoded image invalid. Therefore i thought it would be a good idea to make it explicit and always set it to `Chunky`.